src/views/ipv/page/pyi-update-details.njk

```diff
@@ -0,0 +1,109 @@
+{% set errorText = 'pages.pyiUpdateDetails.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+
+{% block content %}
+    <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}"
+        xmlns="http://www.w3.org/1999/html">{{ 'pages.pyiUpdateDetails.header' | translate }}</h1>
+
+    {% set rows = [
+      {
+        key: {
+          text: 'pages.pyiUpdateDetails.content.summaryListHeading1' | translate
+        },
+        value: {
+          text: userDetails.formattedName.GivenName
+        }
+      },
+      {
+        key: {
+          text: 'pages.pyiUpdateDetails.content.summaryListHeading2' | translate
+        },
+        value: {
+          text: userDetails.formattedName.FamilyName
+        }
+      },
+      {
+          key: {
+              text: 'pages.pyiUpdateDetails.content.summaryListHeading3' | translate
+          },
+          value: {
+              text: userDetails.addresses[0].addressDetailHtml | safe
+          }
+      },
+      {
+          key: {
+            text: 'pages.pyiUpdateDetails.content.summaryListHeading4' | translate
+          },
+          value: {
+            text: userDetails.dateOfBirth | GDSDate
+          }
+      }
+    ] %}
+
+    {{ govukSummaryList({
+      rows: rows
+    }) }}
+
+    <form id="confirmAddressActionForm" action="/ipv/page/{{ pageId }}" method="POST">
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+            {% set checkboxConfig = {
+              name: "nationality",
+              fieldset: {
+                legend: {
+                  text: 'pages.pyiUpdateDetails.content.checkboxes.heading' | translate,
+                  isPageHeading: true,
+                  classes: "govuk-fieldset__legend--m"
+                }
+              },
+              hint: {
+                text: 'pages.pyiUpdateDetails.content.checkboxes.hint' | translate
+              },
+              items: [
+                {
+                  value: "givenNames",
+                  text: 'pages.pyiUpdateDetails.content.checkboxes.checkbox1' | translate,
+                  hint: {
+                    text: 'pages.pyiUpdateDetails.content.checkboxes.checkbox1Hint' | translate
+                  }
+                },
+                {
+                  value: "lastName",
+                  text: 'pages.pyiUpdateDetails.content.checkboxes.checkbox2' | translate
+                },
+                {
+                  value: "address",
+                  text: 'pages.pyiUpdateDetails.content.checkboxes.checkbox3' | translate
+                },
+                {
+                  value: "dateOfBirth",
+                  text: 'pages.pyiUpdateDetails.content.checkboxes.checkbox4' | translate
+                },
+                {
+                  divider: 'pages.pyiUpdateDetails.content.checkboxes.checkbox5Divisor' | translate
+                },
+                {
+                  value: "none",
```

Comment:
Slightly nit-picky I know, but could we change the value to be something like `back` / `cancel` instead of `none`? 

Personally `none` suggests that we are intentionally leaving it empty, instead of it being a valid journey event